From 4e46d191500ebddb8a5d41bcd7e5bc58a4281555 Mon Sep 17 00:00:00 2001
From: Ankit Jain <ankitja@vmware.com>
Date: Tue, 22 Nov 2022 14:28:22 +0000
Subject: [PATCH] iavf: Makefile: added alias for i40evf

- For backward compatiblity, we added alias for i40evf
  in Makefile of iavf-v4.5.3 so that iavf.conf doesn't
  conflict with older version of iavf driver.

Signed-off-by: Ankit Jain <ankitja@vmware.com>
---
 src/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Makefile b/src/Makefile
index 27ce04f..2420ef6 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -105,6 +105,7 @@
 	@echo "Creating /etc/modprobe.d/iavf.conf file ..."
 	mkdir -p "${INSTALL_MOD_PATH}/etc/modprobe.d/"
 	echo "blacklist i40evf"  >  "${INSTALL_MOD_PATH}/etc/modprobe.d/iavf.conf"
+	echo "alias i40evf iavf" >>  "${INSTALL_MOD_PATH}/etc/modprobe.d/iavf.conf"
 
 # Install kernel module files without auxiliary. This target is called by the
 # RPM specfile when  generating binary RPMs, and is not expected to modify
@@ -115,6 +116,7 @@
 	@echo "Creating /etc/modprobe.d/iavf.conf file ..."
 	mkdir -p "${INSTALL_MOD_PATH}/etc/modprobe.d/"
 	echo "blacklist i40evf"  >  "${INSTALL_MOD_PATH}/etc/modprobe.d/iavf.conf"
+	echo "alias i40evf iavf" >>  "${INSTALL_MOD_PATH}/etc/modprobe.d/iavf.conf"
 
 # After installing all the files, perform necessary work to ensure the system
 # will use the new modules. This includes running depmod to update module
